server/src/lib/model/db/migrations/20200406141259-sentence-tagging-structure.ts

```diff
@@ -0,0 +1,42 @@
+export const up = async function(db: any): Promise<any> {
+  return db.runSql(
+    `
+    CREATE TABLE taxonomies (
+      id INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+
+      tax_name VARCHAR(255) NOT NULL,
```

Comment:
Slight nitpick, `tax_name` is harder to guess than `taxonomy_name`. Still going to land this, just food for thought if you think it's worth updating.